secp-ffm/src/test/java/org/bitcoinj/secp/ffm/EcdhTest.java

```diff
@@ -1,7 +0,0 @@
-package org.bitcoinj.secp.ffm;
-
-/**
- *
- */
-public class EcdhTest {
-}
```

Comment:
Oh, and here you're removing it again? Seems like this aspect could be squashed (out of existance).

Reply:
Agreed, I'll do a squash before merging.